Author: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>

Building with fatal warnings prevents xwalk from linking on Tizen Common:

/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: warning: hidden symbol 'sqlite3_close' in obj/third_party/sqlite/libsqlite3.a(obj/third_party/sqlite/amalgamation/sqlite.sqlite3.o) is referenced by DSO /usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../lib64/libpkgmgr_parser.so
/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: warning: hidden symbol 'sqlite3_exec' in obj/third_party/sqlite/libsqlite3.a(obj/third_party/sqlite/amalgamation/sqlite.sqlite3.o) is referenced by DSO /usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../lib64/libpkgmgr_parser.so
/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: warning: hidden symbol 'sqlite3_free' in obj/third_party/sqlite/libsqlite3.a(obj/third_party/sqlite/amalgamation/sqlite.sqlite3.o) is referenced by DSO /usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../lib64/libpkgmgr_parser.so
/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: warning: hidden symbol 'sqlite3_snprintf' in obj/third_party/sqlite/libsqlite3.a(obj/third_party/sqlite/amalgamation/sqlite.sqlite3.o) is referenced by DSO /usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../lib64/libpkgmgr_parser.so
/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: warning: hidden symbol 'sqlite3_close' in obj/third_party/sqlite/libsqlite3.a(obj/third_party/sqlite/amalgamation/sqlite.sqlite3.o) is referenced by DSO /usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../lib64/libpkgmgr-info.so
/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: warning: hidden symbol 'sqlite3_exec' in obj/third_party/sqlite/libsqlite3.a(obj/third_party/sqlite/amalgamation/sqlite.sqlite3.o) is referenced by DSO /usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../lib64/libpkgmgr-info.so
/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: warning: hidden symbol 'sqlite3_free' in obj/third_party/sqlite/libsqlite3.a(obj/third_party/sqlite/amalgamation/sqlite.sqlite3.o) is referenced by DSO /usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../lib64/libpkgmgr-info.so
/usr/lib64/gcc/x86_64-tizen-linux/4.8/../../../../x86_64-tizen-linux/bin/ld: error: treating warnings as errors

This patch disables fatal warnings to allow us to carry on with moving Crosswalk to Chromium 35.

XWALK-1379
--- src/build/common.gypi
+++ src/build/common.gypi
@@ -3082,7 +3082,6 @@
     ['os_posix==1 and (chromeos==0 or target_arch!="arm")', {
       'target_defaults': {
         'ldflags': [
-          '-Wl,--fatal-warnings',
           '-Wl,-z,now',
           '-Wl,-z,relro',
         ],
